test/server.test.js

```diff
@@ -69,3 +70,16 @@ test('listen should reject string port', async (t) => {
     t.equal(error.code, 'ERR_SOCKET_BAD_PORT')
   }
 })
+
+test('listen should not start server if received abort signal', { skip: semver.lt(process.version, '15.6.0') }, t => {
```

Comment:
```suggestion
test('listen should not start server if received abort signal', { skip: semver.lt(process.version, '16') }, t => {
```

Should also do the trick

Reply:
@metcoder95 It cause `TypeError: Invalid Version: 16`

Reply:
Sorry, it should be `16.0.0`